src/ocm/components/clusterConfiguration/staticIp/data/formDataFromInfraEnvField.ts

```diff
@@ -46,6 +47,14 @@ const findFirstRealEthernetInterface = (
   ) as unknown as NmstateEthernetInterface | undefined;
 };
 
+const findFirstVlanInterface = (
+  interfaces: NmstateInterface[],
+): NmstateVlanInterface | undefined => {
+  return interfaces.find((currentInterface) => isVlanInterface(currentInterface)) as unknown as
```

Comment:
I don't think the casting is necessary because `isVlanInterface` already returns an element of type `NmstateVlanInterface`

I think this should work:

```
const findFirstVlanInterface = (
  interfaces: NmstateInterface[],
): NmstateVlanInterface | undefined => {
  return interfaces.find(isVlanInterface);
};
```